samples/mini-games/src/godotMain/kotlin/example/TestingClass.kt

```diff
@@ -37,11 +37,11 @@ class TestingClass : Node() {
     @RegisterSignal
     val signal2 by signal<Boolean, String>("param1", "param2")
 
-    init {
-        signal0.connect(this, this::signal0Receiver)
-        signal1.connect(this, this::signal1Receiver)
-        signal2.connect(this, this::signal2Receiver)
-    }
+//    init {
```

Comment:
this is a no-no, `this` is not initialized yet here. We need to avoid the init block.

Reply:
True. This was really a bad testing sample that was only there to check the fix for #157. Anyway we still need to add that bit to the documentation as preventing it is not possible.